pkg/kubelet/reason_cache.go

```diff
@@ -40,17 +40,18 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// reasonInfo is the cached item in ReasonCache
-type reasonInfo struct {
-	reason  error
-	message string
+// Reason is the cached item in ReasonCache
+type Reason struct {
```

Comment:
You still don't need to export `Reason` since it's only used in this package, right?